File: SmartMoneyApp.Common/DTO/TodoItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SmartApp.Common.DTO
{
    public class TodoItem
    {
        public Int64 Id { get; set; }
        public string Description { get; set; }

        public DateTime DueDate { get; set; }
    }
}

File: SmartApp.Core/Entities/TodoItem.fs
namespace SmartApp.Core.Entities

open System.ComponentModel.DataAnnotations
open System
open System.ComponentModel.DataAnnotations.Schema


    [<CLIMutable>] 
    type  TodoItem   =
        {
            [<Key>]
            [<DatabaseGenerated(DatabaseGeneratedOption.Identity)>]
            Id: int64
            Description: string
            DueDate: DateTime   
        }

